dbcon/joblist/jlf_tuplejoblist.cpp

```diff
@@ -556,6 +558,47 @@ void adjustLastStep(JobStepVector& querySteps, DeliveredTableMap& deliverySteps,
     deliverySteps[CNX_VTABLE_ID] = jobInfo.annexStep;
   }
 
+  //if (orderby_threads == 99)
+  //{
+    SJSTEP acs(new TupleAnnexConstantStep(jobInfo));
+
+   if (jobInfo.constantCol == CONST_COL_EXIST)
+      dynamic_cast<TupleAnnexConstantStep*>(acs.get())->addConstant(new TupleConstantStep(jobInfo));
+
+  if (jobInfo.hasDistinct)
+      dynamic_cast<TupleAnnexConstantStep*>(acs.get())->setDistinct();
```

Comment:
You are not supposed to do distinct in this step.

Reply:
Removed.